src/middlewared/middlewared/pytest/unit/api/base/types/test_base.py

```diff
@@ -48,3 +50,13 @@ class LongStringDefaultMethodArgs(BaseModel):
 
 def test_long_string_default():
     assert accept_params(LongStringDefaultMethodArgs, []) == [""]
+
+
+def test_no_secretstr():
+    result = subprocess.run(
+        ["grep", "-r", "--exclude-dir=pytest", "SecretStr", "/usr/local/lib/python3.11/dist-packages/middlewared/"],
```

Comment:
Great idea! But this is is very brittle. Let’s do the same grep in GitLab CI script after running flake8. Or maybe there even is some flake8 plug-in for prohibited imports?